### DISCUSSION TITLE
How to use Connection to database pane in Positron

### DISCUSSION THREAD

**Discussion by ntluong95** [posit-dev/positron#3806] [Q&A]
Any idea of how to use it. I don't see any button or guidance of how to do it

**Comment by dfalbel** [comment]
Sorry, It's still mostly undocumented. 

If you are using R - the Positron Connections Pane uses the same contract as RStudio, thus it should work with any package that supports the [Rstudio Connections contract](https://rstudio.github.io/rstudio-extensions/connections-contract.html) such as [odbc](https://github.com/r-dbi/odbc), [sparklyr](https://github.com/r-dbi/odbc), etc. Once you open a supported connection type it will automatically populate the connections pane.

For Python, we currently support the sqlite using the `sqlite3` module and `sqlalchemy` connections. Once you create an object, you will see a small database icon close to the object name in the variables pane, if you click there, you'll be able to navigate to the connections pane:

https://github.com/posit-dev/positron/assets/4706822/4c382f7a-e035-408a-8f10-6389d47a97bf

Please feel free to ask additional questions and/or provide feedback :)
Edit: just added an article to the wiki containing a few more details: https://github.com/posit-dev/positron/wiki/Connections-Pane
